tests/rp2040/alire.toml

```diff
@@ -0,0 +1,14 @@
+name = "tests"
+description = "Shiny new project"
+version = "0.0.0"
+
+executables = ["tests"]
+[[depends-on]]
+atomic = "*"
+gnat_arm_elf = "*"
+
+[[pins]]
+atomic = { path='../../../atomic' }
```

Comment:
This should be:
```
atomic = { path='../../' }
```
Otherwise it may not work if the repo is checked-out with a different folder name.

Reply:
I see that I have the same issue in the `armv6m` version I checked-in.

Reply:
Indeed. I had copied the `armv6m` version to ensure that the `rp2040` backend would be consistent with it.

I've updated all of the pins in the `tests` directory in commit 5065f00 to simplify these paths.